ios/Framework/Core/Asset/ListScreenlet/AssetListPageLiferayConnector.swift

```diff
@@ -132,6 +132,20 @@ public class AssetListPageLiferayConnector: PaginationLiferayConnector {
 		return entryQuery
 	}
 
+
+	//MARK: Private methods
+
+	private func handleUserVisibleFlag(values: [String : AnyObject]) -> [String : AnyObject] {
+		if (classNameId == AssetClasses.getClassNameId(AssetClassNameKey_User)) {
+			var newValues = values
+
+			newValues.updateValue("false", forKey: "visible")
+			newValues.updateValue("", forKey: "groupIds")
```

Comment:
- You're right.
- It's not necessarily a `string`. It could be a boolean. I'll change it :)
- Users can only be listed with the `groupId` of the global site. So, i'll remove that line and only change the visible attribute. If you want to list users, you have to put the correct groupId on the screenlet.